services/explorer-ui/src/lib/create-hash-string.ts

```diff
@@ -0,0 +1,5 @@
+export const createHashString = (value: string) => {
```

Comment:
Normally you only do first 6 and last 4. I have examples here: `services/explorer-ui/src/components/contracts/classes/columns.tsx`

Maybe you can also call this function `truncateHashString` and use it everywhere?

Reply:
I see you committed some changes. But not replacing the existing use-cases. Was it intentional? If so, then you can resolve this conversation.